site/README.html

```diff
@@ -0,0 +1,19 @@
+<h1>match-three-js</h1>
+<p>An engine for driving <a href="https://en.wikipedia.org/wiki/Category:Match_3_games">a &quot;match three&quot; game</a>, written in Javascript. Allows for plenty of customization, but defaults to mimic the match three system found in <a href="https://www.youtube.com/watch?v=GGWdqf1BnXo&amp;t=13s"><em>Gems of War</em></a>. This app is meant to be used as a core engine for a match three game, meaning that it contains no business logic concerning turns, or stats. This module only handles the moving, matching, removing, replenishing, and shuffling of &quot;orbs&quot;. Orbs are the units that populate the board, and must be matched in groups of three or more to be removed from the board.</p>
+<h5>Features</h5>
+<p>This list highlights a typical turn in a game of a match three style game. It also conveniently highlights the features of this module.</p>
+<ol>
+<li>Populates an initial board of orbs in a state that has no matches currently, but has at least one potential match.</li>
+<li>Orb values are customizable. They default to integers 0 through 7.</li>
+<li>Board sizes are customizable. They default to 8x8 in size.</li>
+<li>Boards support &quot;swapping&quot;, which move one orb to another adjacent square (not diagonal) to attempt a match.</li>
+<li>Boards scan for matches after swap calls. Any locations on the board that have a consecutive 3+ run of the same orb in a row are &quot;matched&quot;.</li>
+<li>Matched orbs are tallied, and removed. You are given an object informing you of what orbs were removed, and where, for animation purposes.</li>
+<li>Matched orbs are replaced from above with new orbs (gravity defaults to &quot;down&quot;). What type of orbs drop in is random by default, but can be configured.</li>
+<li>After new orbs drop in, more matches are searched for. Any matches found are returned to you for animation purposes.</li>
+<li>This cycle continues until there are no more matches available.</li>
+<li>If there are zero potential matches, the board is shuffled until there are no current board matches, but at least one potential match.</li>
+<li>Control of the board is returned to the consumer.</li>
+</ol>
+<h1>License</h1>
+<p>This application under <a href="./LICENSE">the MIT License</a>.</p>
```

Comment:
Is there a way to ignore this generated file in `master` but keep it checked into the `gh-pages` branch?

I tend to discourage the act of checking in generated content like this, and instead generate the content locally when the project is cloned and started.